### PR TITLE
Prusa3D.PrusaSlicer version 2.4

### PR DESCRIPTION
**Pull request by s-crypt**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/38730)

### CODE REVIEW

File: manifests/p/Prusa3D/PrusaSlicer/2.4/Prusa3D.PrusaSlicer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Prusa3D.PrusaSlicer
+PackageVersion: 2.4
```
Comment (OfficialEsco) [inline manifests/p/Prusa3D/PrusaSlicer/2.4/Prusa3D.PrusaSlicer.installer.yaml:5]: Needs to match `A`dd or `R`emove `P`rograms, remember to change the folder version name :)
```suggestion
PackageVersion: 2.4.0
```

File: manifests/p/Prusa3D/PrusaSlicer/2.4/Prusa3D.PrusaSlicer.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://cdn.prusa3d.com/downloads/drivers/PrusaSlicer_Win_standalone_2.4.0.exe
+  InstallerSha256: C7D1DB2BA3754F9A64DD2E10BEF1F0B6AD28053DE5A1FFF2B0B248D7D505973B
+  InstallerSwitches:
+    Silent: /silent
```
Comment (OfficialEsco) [inline manifests/p/Prusa3D/PrusaSlicer/2.4/Prusa3D.PrusaSlicer.installer.yaml:19]: Someone have been drunk with this switch
```suggestion
    Silent: /quiet
```

File: manifests/p/Prusa3D/PrusaSlicer/2.4/Prusa3D.PrusaSlicer.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Prusa3D.PrusaSlicer
+PackageVersion: 2.4
```
Comment (OfficialEsco) [inline manifests/p/Prusa3D/PrusaSlicer/2.4/Prusa3D.PrusaSlicer.locale.en-US.yaml:5]: Needs to match `A`dd or `R`emove `P`rograms, remember to change the folder version name :)
```suggestion
PackageVersion: 2.4.0
```

File: manifests/p/Prusa3D/PrusaSlicer/2.4/Prusa3D.PrusaSlicer.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Prusa3D.PrusaSlicer
+PackageVersion: 2.4
```
Comment (OfficialEsco) [inline manifests/p/Prusa3D/PrusaSlicer/2.4/Prusa3D.PrusaSlicer.yaml:5]: Needs to match `A`dd or `R`emove `P`rograms, remember to change the folder version name :)
```suggestion
PackageVersion: 2.4.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
You can fix the version folder name with GitHub Desktop or via VSCode Online by pressing `.`

**Comment by s-crypt** [comment]
@OfficialEsco The bot didn't like the manifest change :\

**Comment by OfficialEsco** [comment]
Yeah cuz you have to change the version folder name
```
Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
actual 'manifests\p\Prusa3D\PrusaSlicer\2.4', 
expected 'manifests\p\Prusa3D\PrusaSlicer\2.4.0'
```

**Comment by s-crypt** [comment]
haha whoops.

**Review by OfficialEsco — APPROVED** [review]
Great, thanks =)

**Pull request merged**